pkg/proxy/fixtures/gencerts.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
+set -euo pipefail
+
 if ! [[ "$0" =~ "./gencerts.sh" ]]; then
   echo "must be run from 'fixtures'"
   exit 255
 fi
 
-if ! which cfssl; then
+if ! command -v cfssl; then
   echo "cfssl is not installed"
+  echo "use: go install -mod mod github.com/cloudflare/cfssl/cmd/cfssl"
```

Comment:
FYI. `-mod=mod tells the go command to ignore the vendor directory and to [automatically update](https://go.dev/ref/mod#go-mod-file-updates) go.mod, for example, when an imported package is not provided by any known module.`. See https://go.dev/ref/mod#build-commands.

This comment applies to a couple of places below.

```suggestion
  echo "use: go install github.com/cloudflare/cfssl/cmd/cfssl"
```

Reply:
You're right. It will dirty the go.mod if the user runs the command in the repo. I updated with your suggestion. Please take a look. Thanks!

Reply:
Please consider to install a specific version of tool (e.g. cfssl in this case). 

Usually we specify the version in tools/mod:
1. import the package in tools/mod/tools.go;
2. specify the version in tools/mod/go.mod;
3. Execute `cd tools/mod; go install github.com/cloudflare/cfssl/cmd/cfssl`

This can be resolved in a separate PR.

Reply:
Sure! That makes sense. Will file other pr later.